gtdynamics/universal_robot/Joint.cpp

```diff
@@ -23,16 +23,16 @@
 namespace gtdynamics {
 
 /* ************************************************************************* */
-Joint::Joint(uint8_t id, const std::string &name, const Pose3 &wTj,
+Joint::Joint(uint8_t id, const std::string &name, const Pose3 &bTj,
              const LinkSharedPtr &parent_link, const LinkSharedPtr &child_link,
              const JointParams &parameters)
     : id_(id),
       name_(name),
       parent_link_(parent_link),
       child_link_(child_link),
       parameters_(parameters) {
-  jMp_ = wTj.inverse() * parent_link_->wTcom();
-  jMc_ = wTj.inverse() * child_link_->wTcom();
+  jMp_ = bTj.inverse() * parent_link_->bTcom();
```

Comment:
This could be written as `bTj.between(parent_link_->bTcom())`, but maybe it is more clear as is…

Reply:
I think it is more clear this way